src/components/App/App.vue

```diff
@@ -77,14 +77,19 @@ export default {
       if (foundLocation) {
         const foundLocationIndex = this.locations.indexOf(foundLocation)
         this.locations[foundLocationIndex] = newLocation
-      } else {
+      } else if (!newLocation.message) {
```

Comment:
Not 100% sure if this fixed it. 'message' isn't showing up in local storage anymore but my errors are still persisting in the console so I think it may be fixed but just difficult to actually test.

Reply:
I'm not seeing the message in LS either, and not getting console errors, but am still seeing LS keeping cards that I had tried to delete, so those are reappearing on a page refresh

Reply:
Actually, on a re-test, for some reason that only seems to apply to the NEAREST LOCATION card. I can successfully delete other locations and they do not return on refresh.
I can delete the nearest location card and see it removed from LS, but that location/card alone is added back to local storage and displayed again on page refresh.